packages/e2e/test/wallet/PersonalWallet/phase2validation.test.ts

```diff
@@ -59,7 +59,10 @@ describe('PersonalWallet/phase2validation', () => {
     wallet.shutdown();
   });
 
-  it('can detect a phase2 validation failure and emit the transaction as failed', async () => {
+  // The submit fails instead of being able to send it.
+  // TODO: re-enable this after understanding how to force submit API to submit a transaction that
+  // will fain in phase2 validation
+  it.skip('can detect a phase2 validation failure and emit the transaction as failed', async () => {
```

Comment:
I would say this is another unexpected good news!
I see this as a new added feature for free: now users can know about a phase 2 validation failure without spending fee and collaterals.
On the other side it's harder for us to test this still possible case; probably we need a smart contract accepting only one tx, sending to it two txs simultaneously one is expected to fail phase 2 validation... (just brainstorming)

Reply:
I would like to have also @mkazlauskas and @AngelCastilloB opinions